lib/data/external/GCP/GcpManagedUpload.js

```diff
@@ -65,80 +68,57 @@ class GcpManagedUpload {
      */
     validateBody() {
         this.body = this.params.Body;
-        if (!this.body) {
-            this.error = errors.InvalidArgument
-                .customizeDescription('Body parameter is required');
+        assert(this.body, errors.MissingRequestBodyError.customizeDescription(
```

Comment:
An `AssertionError` should suffice. However, I am wondering what impact that these assertions will have. Would this not effectively stop the server when we want to instead log an error and return an error? @jonathan-gramain 

Maybe I'm mistaken about that, but was wondering because you removed the test for this case.

Reply:
Yes. the assertion does stop the server.

Reply:
Have made the necessary changes to wrap the construction of the GcpManageUpload constructor in a try/catch. If an error is thrown, the callback will be called with the returned error.